src/sentry/static/sentry/app/components/events/contextSummary.jsx

```diff
@@ -56,6 +56,51 @@ class GenericSummary extends React.Component {
   }
 }
 
+export class OsSummary extends React.Component {
+  static propTypes = {
+    data: PropTypes.object.isRequired,
```

Comment:
This is out of scope for this PR, but it'd be great if we could use `PropTypes.shape` to describe the object that is expected for all of these summaries. We currently have `app/proptypes` for an event, but not all of the keys are flushed out.

Reply:
Sounds good!